nornir/core/task.py

```diff
@@ -131,6 +143,33 @@ def is_dry_run(self, override: bool = None) -> bool:
         """
         return override if override is not None else self.nornir.data.dry_run
 
+    def get_connection(self, conn_plugin: Type["ConnectionPlugin"]):
```

Comment:
I don't think this belongs here. The task is not connected to a device, a `Host` is connected to a device.